features/issues/components/issue-list/issue-filter.tsx

```diff
@@ -3,46 +3,63 @@ import { Select, Input, StyledButton } from "@features/ui";
 import { SelectState } from "@features/ui/select/select";
 import { InputIcon, InputLabel, InputState } from "@features/ui/input/input";
 import { ButtonIcon } from "@features/ui/button/button";
+import { breakpoint } from "@styles/theme";
 import { useState } from "react";
-import { useSelector, useDispatch } from "react-redux";
 
 const Container = styled.div`
   display: flex;
   flex-direction: column-reverse;
   align-items: center;
   justify-content: space-between;
   margin-bottom: 25px;
+  & > button {
+    width: 100%;
+  }
+
+  @media (min-width: ${breakpoint("desktop")}) {
+    flex-direction: row;
+    & > button {
+      width: inherit;
+    }
+  }
 `;
 
 const FilterContainer = styled.div`
   display: flex;
+  flex-direction: column;
   width: 100%;
+
+  @media (min-width: ${breakpoint("desktop")}) {
+    flex-direction: row;
+    align-items: center;
+    width: 70%;
+  }
 `;
 
 const BoxContainer = styled.div`
-  width: 160px;
-  height: 44px;
   margin-right: 16px;
   margin-bottom: 16px;
   width: 100%;
+
+  @media (min-width: ${breakpoint("desktop")}) {
+    margin-bottom: 0px;
+  }
 `;
 
 export const IssueFilter = () => {
   const [statusSelectOpen, setStatusSelectOpen] = useState(false);
   const [levelSelectOpen, setLevelSelectOpen] = useState(false);
-  const [currentState, setCurrentState] = useState("All status");
-  const [currentLevel, setCurrentLevel] = useState("All level");
   const stateSelectOptions = ["All status", "Unresolved", "Resolved"];
   const levelSelectOptions = ["All level", "Error", "Warning", "Info"];
-  const issues = useSelector((state: any) => state.issues);
 
   return (
     <Container>
-      <StyledButton icon={ButtonIcon.leading} fill={true}>
+      <StyledButton icon={ButtonIcon.leading}>
         Resolve selected issues
       </StyledButton>
+
       <FilterContainer>
-        <BoxContainer>
+        <BoxContainer className="status-container">
```

Comment:
This doesn't seem to be used. Fyi to style a custom element that accepts a className prop you can also wrap it in a styled component

```
styled(BoxContainer)`...`
```